compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Dialog.skiko.kt

```diff
@@ -166,14 +214,27 @@ private fun DialogLayout(
     onOutsidePointerEvent: ((eventType: PointerEventType, button: PointerButton?) -> Unit)? = null,
     content: @Composable () -> Unit
 ) {
+    val layer = rememberComposeSceneLayer(focusable = true)
+    layer.setOutsidePointerEventListener(onOutsidePointerEvent)
     val currentContent by rememberUpdatedState(content)
+    val graphicsContext = LocalGraphicsContext.current
+
+    val animator = remember {
```

Comment:
It probably should be like this
```suggestion
    val animator = remember(layer, graphicsContext, properties.animateTransition) {
```

What's the current behavior in case of re-compositing with a changed layer or flag?

Reply:
As animator is remembered by the `LaunchedEffect` and `DisposableEffect`, adding these parameters won't make the situation better.
Currently, the behavior does not expect any of these 3 parameters to be changed. If they can or should - we have to reconsider he entire approach here.

Reply:
`properties.animateTransition` - "should" as it is a public parameter
others probably can, but I'm not sure

Reply:
Implemented, but without interrupting ongoing animations